app/lib/dfe_sign_in_api/get_open_organisations_for_user.rb

```diff
@@ -0,0 +1,27 @@
+module DfESignInApi
+  class GetOpenOrganisationsForUser
```

Comment:
Maybe 'GetOrganisationsForUser' (or something that doesn't explicitly say 'open') would be better as there are quite a few states that aren't 'closed'. e.g. 'opening', 'closing' that we're including here.